README.md

```diff
@@ -50,7 +52,9 @@ Correctly set test.properties and credentials.json (described later)
 
 Added FUSE repositories to maven due to dependencies
 
-Before you import maven project to the IDE, you have to installed *Lombok* and *Cucumber* plugins to the IDE.
+Before you import maven project to the IDE, you have to installed 
+**[Lombok](https://plugins.jetbrains.com/plugin/6317-lombok-plugin)** and 
```

Comment:
I'd mark this list that it applies for Idea. If someone uses anything else, they can add a section for IDE of their choice.